src/Tools/BuildValidator/BuildConstructor.cs

```diff
@@ -59,53 +62,115 @@ public Compilation CreateCompilation(MetadataReader metadataReader, string name)
             throw new InvalidDataException("Did not find language in compilation options");
         }
 
-        private ImmutableArray<MetadataReference> CreateMetadataReferences(CompilationOptionsReader pdbReader)
+        private ImmutableArray<MetadataReferenceInfo> GetMetadataReferenceInfos(CompilationOptionsReader compilationOptionsReader)
```

Comment:
Nit: could make this a simple call to `GetMetadataReferences` vs. storing into a local here.

Reply:
Fixed